### ISSUE TITLE
Project and connection management

### ISSUE THREAD

**Issue by dlucazeau** [mingo-app/mingo#425]
The 'project' concept is a good idea, but it lacks the ability to move a connection from one project to another or to attach a connection without a project to a project.

**Comment by tothradoslav** [comment]
Yep, that's true. Do you have any ideas on how this should be done, ideally? We'll think of a solution, too :)

**Comment by dlucazeau** [comment]
Perhaps, a first step can be with a drag and drop. And in a second step, a drop-down on the connection details.

**Comment by tothradoslav** [comment]
Ok, the first change we made is that in a project, when you click "Add database", it gives you a context menu with options to either add a blank new connection or select a database from all the current "standalone" connections (connections without a project) and projects, too. This is a big UX improvement, thank you!

**Comment by tothradoslav** [comment]
V1.9.0-rc1 is available. Please try it out and please let us know whether your issue has been resolved. Thank you! https://github.com/mingo-app/mingo/releases/tag/v1.9.0-rc1

**Comment by dlucazeau** [comment]
Yes, that is a good correction.
After adding an existing connection, the blank input field should no longer cause an error. It's not very important, we just need to delete it, but it's weird to delete something that doesn't exist :smile:
Thank you.
